### ISSUE TITLE
Cropped picture clarity is so poor

### ISSUE THREAD

**Issue by SajithaMadhav** [ArthurHub/Android-Image-Cropper#886]
Cropped image is blurred one. Any solution to solve this?

**Comment by cxp-13** [comment]
no

**Comment by SajithaMadhav** [comment]
Yes, the issue was in my implementation side. I fixed it by setting correct width and height required for me in pixels.

**Comment by Canato** [comment]
please read https://github.com/ArthurHub/Android-Image-Cropper/issues/875

And update to the new repository

Please close this issue.
So the Warning can be more visible

**Comment by alvarezjosejimmy-ship-it** [comment]
> La imagen recortada está borrosa. ¿Alguna solución?
